### PR TITLE
About leadership

### LINKED ISSUE #172: About Us Page "Leadership"
This can definitely be split up into multiple sub components because there's a lot of repeat.

Screen Shot 2019-03-29 at 11 38 48 AM

### PR DESCRIPTION
**Pull request by Stephanie-A**
Basically has a bunch of board member descriptions continuing after closes #172 

Expected:
image

Actual:
image

### CODE REVIEW

File: client/src/components/BoardOfDirectors.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import './css/BoardOfDirectors.css';
+
+class BoardOfDirectors extends Component {
```
Comment (SujeethJinesh) [inline client/src/components/BoardOfDirectors.js]: Add comments to the top of these files

**Comment by SujeethJinesh** [comment]
Looks good, just need to add some comments!

File: client/src/components/BoardOfDirectors.js (hunk 1 above)
Comment (Stephanie-A) [reply client/src/components/BoardOfDirectors.js]: What would the comments be for/of?

Comment (SujeethJinesh) [reply client/src/components/BoardOfDirectors.js]: just a small description of what the component's job is, where it's located on the frontend, and any things you deem necessary. Also any TODOs

File: client/src/components/BoardOfDirectors.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<<<<<<< HEAD
+//component for descriptions of each member of the board of directors
+=======
+>>>>>>> 465aaa67c26e82e3d845e1acedfc33090b79fc86
```
Comment (SujeethJinesh) [inline client/src/components/BoardOfDirectors.js]: Oopsies

also for the comments, i meant something like

```
/**
 * Component for descriptions of each member of the board of directors
*/
class BoardOfDirectors ...
```

Comment (SujeethJinesh) [reply client/src/components/BoardOfDirectors.js]: I meant on top of the class instead of the top of the file, my b!

Comment (SujeethJinesh) [reply client/src/components/BoardOfDirectors.js]: I'll take care of this, so no worries.

**Pull request merged**